crates/forge/bin/cmd/create.rs

```diff
@@ -867,7 +862,47 @@ where
 
         Ok((address, receipt))
     }
+}
 
+/// Helper which manages the deployment transaction of a smart contract
+#[derive(Debug)]
+#[must_use = "Deployer does nothing unless you `send` it"]
+pub struct ZkDeployer<B, P, T> {
+    /// The deployer's transaction, exposed for overriding the defaults
+    pub tx: alloy_zksync::network::transaction_request::TransactionRequest,
+    abi: JsonAbi,
+    client: B,
+    confs: usize,
+    timeout: u64,
+    zk_factory_deps: Option<Vec<Vec<u8>>>,
+    _p: PhantomData<P>,
+    _t: PhantomData<T>,
```

Comment:
```suggestion
    _pt: PhantomData<(P, T)>,
```

Reply:
This mirrors upstream variant. Keeping it consistent.